_data/alerts.yml

```diff
@@ -4,12 +4,12 @@ important: '<div class="alert alert-warning" role="alert"><i class="fa fa-warnin
 warning: '<div class="alert alert-danger" role="alert"><i class="fa fa-exclamation-circle"></i> <b>Warning: </b>'
 end: '</div>'
 
-callout_danger: '<div class="bs-callout bs-callout-danger">'
-callout_default: '<div class="bs-callout bs-callout-default">'
-callout_primary: '<div class="bs-callout bs-callout-primary">'
-callout_success: '<div class="bs-callout bs-callout-success">'
-callout_info: '<div class="bs-callout bs-callout-info">'
-callout_warning: '<div class="bs-callout bs-callout-warning">'
+callout_danger: '<div class="bs-callout bs-callout-danger" markdown="1">'
```

Comment:
Oh I didn't know you had to add this stuff :( Thanks for figuring that out though.